scripts/salt-install.sh

```diff
@@ -71,6 +71,8 @@ function update_yum_repos() {
     REPO_FILE=rhel${RHEL_VERSION}_cldr_mirrors.repo
     rm /etc/yum.repos.d/*.repo -f
     curl https://mirror.infra.cloudera.com/repos/rhel/server/8/${RHEL_VERSION}/${REPO_FILE} > /etc/yum.repos.d/${REPO_FILE}
+    # remove whitespaces around '=' as CIS checks do not handle those
```

Comment:
As it is a result of a CIS rule, It would make more sense to have it in the common.sls in a generic way to apply the replace for each repo files.